### PR TITLE
Update run-test.sh to follow up recent change

### PR DESCRIPTION
**Pull request by hqueue**
Because layout of CoreFX test has been changed,
run-test.sh should be updated too.

### CODE REVIEW

**Comment by hqueue** [comment]
Also please let me know which PR has changed the layout of CoreFX tests.

There was no `bin/tests` before when build from Linux/x64, 
but now built tests are moved to `bin/tests` directory.

**Comment by hqueue** [comment]
@dotnet-bot please tset Linux arm Release Build

**Comment by hqueue** [comment]
@dotnet-bot please test Linux arm Release Build

**Comment by stephentoub** [comment]
@weshaggard, I assume the change came from buildtools?

**Comment by weshaggard** [comment]
@safern does this map to what your test output changes look like?

**Comment by safern** [comment]
Yes. https://github.com/dotnet/corefx/pull/23633#issue-253522995

I think this if can also be safely removed: https://github.com/dotnet/corefx/pull/23942/files#diff-b85dc439de2f53c86324409385ec0b5aR201 as there is no netstandard runtime and it will always be netcoreapp. 

Sorry for not updating this when updating buildtools.

**Comment by safern** [comment]
test UWP CoreCLR x64 Debug Build please

File: run-test.sh [hunk 1]
```diff
@@ -197,8 +197,7 @@ run_test()
     exit 0
   fi
 
-  dirName="$1/netcoreapp"
-
+  dirName="$1/netcoreapp-$OS-$ConfigurationGroup-$__Arch"
   if [ ! -d "$dirName" ]; then
```
Comment (weshaggard) [inline run-test.sh]: You should be able to remove this netstandard fallback logic.

Comment (hqueue) [reply run-test.sh]: Thanks! I've update the PR.

**Comment by hqueue** [comment]
@stephentoub @weshaggard @safern Thanks for the comment I've update the PR.

**Pull request merged**